### ISSUE TITLE
Don't print errors in MDX files

### ISSUE THREAD

**Issue by hasparus** [cometkim/gatsby-plugin-typegen#117]
Howdy 👋

I just encountered this fellow 

> [typegen] Fail to extract GraphQL documents from /home/hasparus/workspace/homepage/src/pages/music.mdx Provided file type must be one of .js, .jsx, .ts, .tsx, .flow, .flow.js, .flow.jsx, .vue
>
> TypeError: Provided file type must be one of .js, .jsx, .ts, .  tsx, .flow, .flow.js, .flow.jsx, .vue
 
We couldn't use a generated type in `.mdx` file regardless, so this error seems redundant.

Is there a way to specify file extensions in plugin options, so the error doesn't happen?

**Comment by cometkim** [comment]
Hmm, I'm confusing. this plugin is notified by Gatsby what files have actually proceeded.

@hasparus Do you have GraphQL document snippets in the mdx file?
Could you share your project config or provide a minimal repro?

**Comment by hasparus** [comment]
yup, I have a pageQuery in MDX file

```mdx

some **markdown**

<Component site={page.data.sitePage} />

export const pageQuery = graphql`
  query MusicPageQuery($path: String!) {
    sitePage(path: {eq: $path}) {
      ...TweetDiscussEditLinksDataOnSitePage
    }
  }
`
```

I'm not sure if my config will be helpful, but [I have it public on GitHub right here](https://github.com/hasparus/homepage/blob/54f51c4ed22cbd46f57b1f22e16d3b840c1a3c0f/gatsby-config.ts#L49-L64).
I'm using `gatsby-plugin-mdx` with an MDX page inside of `/pages`.

**Comment by cometkim** [comment]
This should be fixed in v3 (gatsby-plugin-typegen@3.0.0-rc.0)
